### PR TITLE
Change web files back to var/lib/netdata/www

### PR DESCRIPTION
**Pull request by tkatsoulas**
##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1744,7 +1744,7 @@ set(CONFIG_DIR "${CMAKE_INSTALL_PREFIX}/etc/netdata")
 set(LIBCONFIG_DIR "${CMAKE_INSTALL_PREFIX}/usr/lib/netdata/conf.d")
 set(LOG_DIR "${CMAKE_INSTALL_PREFIX}/var/log/netdata")
 set(PLUGINS_DIR "${CMAKE_INSTALL_PREFIX}/usr/libexec/netdata/plugins.d")
-set(WEB_DIR "${CMAKE_INSTALL_PREFIX}/usr/share/netdata/web")
+set(WEB_DIR "${CMAKE_INSTALL_PREFIX}/var/lib/netdata/www")
```
Comment (ilyam8) [inline CMakeLists.txt:1747]: /var/lib/netdata/www is Debian native only

Comment (tkatsoulas) [reply CMakeLists.txt:1747]: so you mean this will affect also other distros, and we don't want this?

Comment (tkatsoulas) [reply CMakeLists.txt:1747]: we also do this 
https://github.com/netdata/netdata/blob/master/contrib/debian/rules#L204-L208
for debian, let me check side to side the packages produced for both rpm and deb

Comment (ilyam8) [reply CMakeLists.txt:1747]: debian native was the only place where we set web dir (see `--with-webdir`)

https://github.com/netdata/netdata/blob/5d998b946fedddbde106309d57b64ec96f596332/contrib/debian/rules#L43

**Pull request closed without merging**